File: September20th/ex31.c
#include <stdio.h>
/* André Cavalcante de Lima
31. Exibir os vinte primeiros valores da série de Bergamaschi.
 A série: 1, 1, 1, 3, 5, 9, 17, ...

*/

int main()
{
      float a=-1, b=1, c= 1;

      printf("Sequencia dos 20 primeiro do Bergamaschi e: \n");
  
      for(int i=0 ; i<20 ; i++){  
         int r = a+b+c;
          c=b;
          b=a;
          a=r;
                
     printf("%i\n",r);
      }  
    
   return 0;
}

File: October25thPython-17-32/ex31.py
'''/* André Cavalcante de Lima
31. Exibir os vinte primeiros valores da série de Bergamaschi.
 A série: 1, 1, 1, 3, 5, 9, 17, ...

*/'''



a=-1
b=1
c= 1

print("Sequencia dos 20 primeiro do Bergamaschi e: \n");
  
for i in range(0 , 20, 1):
    r = a+b+c
    c=b
    b=a
    a=r
                
    print("",r);
        
    
